crossteam-project-valuation.md

```diff
@@ -0,0 +1,101 @@
+# Title
+
+Cross-team Project Valuation
+
+# Patlet
+
+It's hard to sell the value of cross-team, inner sourced projects that don't provide a direct impact on company revenue.
+Here's a data-driven way to represent your project that both articulates its value and amplifies it.
+
+# Context
+
+* You are responsible for a cross-team that serves as a platform for others in the company.
+* The cross-team project does not deliver any direct value for company revenue.
+
+# Problem
+
+Cross-team projects can potentially have a very large impact on the company yet are notoriosly-difficult to represent in a data-driven fashion.
+As a result, it is easy and common to either pursue projects that does not provide real value or to underfund what would otherwise produce great value.
+Setting in place a pattern and model of how to value such projects gives us the tool that we need to focus and amplify high-value collaboration for the company.
```

Comment:
Should probably be reformulated as a problem  (absence of such a model) or moved to the solution section.

Reply:
Removed from here and moved to to the solution.